### DISCUSSION TITLE
Trouble setting up

### DISCUSSION THREAD

**Discussion by Thomasjohnlyons** [oleghnidets/OHMySQL#30] [Q&A]
Hello! I have a Swift project that I am trying to connect to a MySQL database, and instead of using the whole JSON, PHP script route, this looked like a good solution to connect to it directly. The problem however is I am rather new to all of this. I just figured cocoa pods out, and am trying to set up the demo solution. I have the server running, but when looking in the example project, under the OHSettingsViewController where is has "@import OHMySQL;" I am getting "Module OHMySQL not found". I am really not sure where to go from here please help! Thanks so much for your work.

**Comment by oleghnidets** [comment]
Hi there,
Did you execute `pod install`?

**Reply by Thomasjohnlyons** [reply]
Hello! Yes I did pod update and pod install, I will give that another try though and make sure everything in the podfile looks good.

**Reply by Thomasjohnlyons** [reply]
The other error I get is:  _'OHMySQLMappingProtocol.h' file not found_
